### PR TITLE
installing ledger on windows using wsl updates. Fixes #27

### LINKED ISSUE #27: Is WSL an easier way in 2023 to run ledger?
> https://github.com/rolfschr/GSWL-book/blob/dd833b7531730a2d29553d0dd05e8dbe009b64bd/02-ledger-accounting/01-double-accounting.md?plain=1#L163

This refers to VirtualBox and I wonder if WSL2 might be an easier way to run ledger. Let me know if you want me to send a pull request on usage of WSL2.

### PR DESCRIPTION
**Pull request by jayrajput**

### CODE REVIEW

**Comment by jayrajput** [comment]
Fixes #27

File: 09-end/01.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ The following people helped getting this book finished and/or up to date:
 * Samuel Marcaille (improved html version)
 * Simon Michael (general feedback & proof reading)
 * Trannie Carter (epub version)
+* Jay Rajput (improved html version)
```
Comment (rolfschr) [inline 09-end/01.md]: Could you add your name in alphabetically order?

File: 02-ledger-accounting/01-double-accounting.md [hunk 2]
```diff
@@ -164,29 +164,10 @@ $ brew install ledger
...
-Step-by-step Instructions (with Vagrant):
-
-* Download and install [VirtualBox](https://www.virtualbox.org/) & [Vagrant](https://www.vagrantup.com/).
-* Download this [Vagrantfile](https://github.com/rolfschr/GSWL-ecosystem/blob/master/contrib/Vagrantfile) from Github.
-* Open a terminal, go to the Vagrantfile's location and run ``vagrant up`` (this will setup an Ubuntu machine with Ledger installed).
-* To connect to the VM via SSH, use ``vagrant up`` followed by ``vagrant ssh`` from within the same folder.
+I therefore recommend to run Linux on Windows using WSL (Windows Subsystem for Linux). 
+You can install Linux/Ubuntu by running ``wsl --install`` from PowerShell or Windows Command Prompt in administrator mode.
+Refer to the [WSL Documentation](https://learn.microsoft.com/en-us/windows/wsl/) for more details.
+After WSL installation, follow the steps in Linux section for ledger installation.
```
Comment (rolfschr) [inline 02-ledger-accounting/01-double-accounting.md:170]: :+1:

**Review by rolfschr — APPROVED** [review]
Thanks a bunch for your initiative!

**Comment by jayrajput** [comment]
So what happens now as you have approved the changes. Can I close this pull request?

**Comment by rolfschr** [comment]
Thanks for your contribution!

**Pull request merged**